Makefile

```diff
@@ -34,42 +34,45 @@ help:
 	@echo
 
 
-.PHONY:
+.PHONY: install-requirements
 install-requirements:
 	pipx install tqdm black flake8 argcomplete wheel omlmd huggingface_hub codespell
 
-.PHONY:
```

Comment:
Seems like you are missing a space here

`.PHONY:install-completions`

to

`.PHONY: install-completions`

or at least all other versions have a space.